src/app/cheat-sheets/game-base/space-age/space-age.component.html

```diff
@@ -140,8 +145,8 @@ <h5>Planet Start</h5>
       </li>
 
       <li>
-        Space is can be limited, and scrap recycling yield a lot of products, thus using robots can be very convenient to sort everything
-        out, and craft starter items
+        Space is almost always limited and scrap recycling yields a lot of products, thus using robots can be very convenient to sort everything
+        out and craft starter items.
```

Comment:
```suggestion
        Most islands on Fulgora are small, and recycling scrap gives you a lot of different items. You can use robots to sort the output. Use the items you get from recycling to craft starter items.
```

It would be nice to have a proper explanation of _how_ to sort the items with bots, and how to recycle unwanted items (things you have too much of!). I still haven't figured out a good way to do this with bots, but others probably have. 😉